Pre-compiled binary seg faults

The binary provided inside the zip file seg faults. This has been tried on multiple Linux systems.

```
./opera_Linux
Segmentation fault
```

In addition compilation of the software fails even though the dependencies have been installed. The error 

```
git clone https://github.com/wuyangf7/OPERA
cd OPERA
make
Cloning into 'OPERA'...
remote: Enumerating objects: 586, done.
remote: Counting objects: 100% (109/109), done.
remote: Compressing objects: 100% (57/57), done.
remote: Total 586 (delta 67), reused 81 (delta 51), pack-reused 477
Receiving objects: 100% (586/586), 64.56 MiB | 1.50 MiB/s, done.
Resolving deltas: 100% (329/329), done.
g++ -w -O3 -m64 -fopenmp -I /home/uqywu16/90days/program/lib/boost_1_61_0 -I  -DEIGEN_NO_DEBUG -std=c++11  -c SMR.cpp
g++ -w -O3 -m64 -fopenmp -I /home/uqywu16/90days/program/lib/boost_1_61_0 -I  -DEIGEN_NO_DEBUG -std=c++11  -c CommFunc.cpp
g++ -w -O3 -m64 -fopenmp -I /home/uqywu16/90days/program/lib/boost_1_61_0 -I  -DEIGEN_NO_DEBUG -std=c++11  -c SMR_data.cpp
g++ -w -O3 -m64 -fopenmp -I /home/uqywu16/90days/program/lib/boost_1_61_0 -I  -DEIGEN_NO_DEBUG -std=c++11  -c dcdflib.cpp
g++ -w -O3 -m64 -fopenmp -I /home/uqywu16/90days/program/lib/boost_1_61_0 -I  -DEIGEN_NO_DEBUG -std=c++11  -c StatFunc.cpp
g++ -w -O3 -m64 -fopenmp -I /home/uqywu16/90days/program/lib/boost_1_61_0 -I  -DEIGEN_NO_DEBUG -std=c++11  -c StrFunc.cpp
g++ -w -O3 -m64 -fopenmp -I /home/uqywu16/90days/program/lib/boost_1_61_0 -I  -DEIGEN_NO_DEBUG -std=c++11  -c SMR_plot.cpp
g++ -w -O3 -m64 -fopenmp -I /home/uqywu16/90days/program/lib/boost_1_61_0 -I  -DEIGEN_NO_DEBUG -std=c++11  -c SMR_data_p1.cpp
g++ -w -O3 -m64 -fopenmp -I /home/uqywu16/90days/program/lib/boost_1_61_0 -I  -DEIGEN_NO_DEBUG -std=c++11  -c SMR_data_p2.cpp
g++ -w -O3 -m64 -fopenmp -I /home/uqywu16/90days/program/lib/boost_1_61_0 -I  -DEIGEN_NO_DEBUG -std=c++11  -c stat.cpp
g++ -w -O3 -m64 -fopenmp -I /home/uqywu16/90days/program/lib/boost_1_61_0 -I  -DEIGEN_NO_DEBUG -std=c++11  -c bfile.cpp
g++ -w -O3 -m64 -fopenmp -I /home/uqywu16/90days/program/lib/boost_1_61_0 -I  -DEIGEN_NO_DEBUG -std=c++11  -c SMR_data_p3.cpp
g++ -w -O3 -m64 -fopenmp -I /home/uqywu16/90days/program/lib/boost_1_61_0 -I  -DEIGEN_NO_DEBUG -std=c++11  -o opera_Linux SMR.o CommFunc.o SMR_data.o dcdflib.o StatFunc.o StrFunc.o SMR_plot.o SMR_data_p1.o SMR_data_p2.o stat.o bfile.o SMR_data_p3.o -static -lz -lgomp 
/usr/bin/ld: /usr/lib/gcc/x86_64-linux-gnu/11/libgomp.a(target.o): in function `gomp_target_init.part.0':
(.text+0x358): warning: Using 'dlopen' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/bin/ld: SMR.o: in function `option(int, char**)':
SMR.cpp:(.text+0x28ca): undefined reference to `SMRDATA::multioutcomesmr(char*, char*, char*, char*, char*, char*, char*, char*, double, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, char*, char*, char*, char*, char*, bool, double, double, int, int, char*, char*, char*, char*, char*, double, char*, bool, int, int, int, char*, int, char*, char*, char*, char*, double, bool, bool, double, bool, bool, double, int, char*, char*, double, double)'
/usr/bin/ld: SMR.cpp:(.text+0x9a8a): undefined reference to `SMRDATA::multiexposurepi_jointsmr(char*, char*, char*, char*, char*, double, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, double, double, char*, char*, char*, char*, char*, bool, double, double, int, int, char*, char*, char*, char*, char*, double, char*, bool, bool, int, int, int, int, char*, int, char*, char*, char*, char*, double, bool, bool, double, bool, bool, double, int, char*, char*, char*, double, double)'
/usr/bin/ld: SMR.cpp:(.text+0x9d2f): undefined reference to `SMRDATA::multiexposure_jointsmr(char*, char*, char*, char*, char*, char*, char*, char*, double, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, double, char*, char*, char*, char*, char*, bool, double, double, int, int, char*, char*, char*, char*, char*, double, double, double, double, double, double, char*, bool, bool, bool, bool, bool, int, int, int, char*, int, char*, char*, char*, char*, double, bool, bool, double, bool, bool, double, int, char*, char*, char*, double, double)'
/usr/bin/ld: SMR.cpp:(.text+0x9dd2): undefined reference to `SMRDATA::ppafile_summary(char*, char*, char*, char*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, char*, double, double)'
collect2: error: ld returned 1 exit status
make: *** [Makefile:57: opera_Linux] Error 1
```

```
apt list intel-mkl libboost-dev libeigen3-dev
Listing... Done
intel-mkl/jammy,now 2020.4.304-2ubuntu3 amd64 [installed]
libboost-dev/jammy,now 1.74.0.3ubuntu7 amd64 [installed]
libeigen3-dev/jammy,now 3.4.0-2ubuntu2 all [installed]
```
Could you please provide a working pre-complied static binary?

Thank you

--- a/Makefile
+++ b/Makefile
@@ -24,31 +24,33 @@ CXXFLAGS = -w -O3 -m64 -fopenmp -I $(BOOST) -I $(EIGEN_PATH) -DEIGEN_NO_DEBUG -s
 LIB += -static -lz -lgomp
 
 HDR += CommFunc.h \
-	   cdflib.h \
-	   dcdflib.h \
-           SMR.h \
-	   ipmpar.h \
-           StatFunc.h \
-           StrFunc.h \
-            SMR_data.h \
-            SMR_plot.h \
-            SMR_data_p1.h \
-            SMR_data_p2.h \
-	   stat.hpp \
-	   bfile.hpp \
-           SMR_data_p3.h 
+			 cdflib.h \
+			 dcdflib.h \
+       SMR.h \
+	     ipmpar.h \
+       StatFunc.h \
+       StrFunc.h \
+       SMR_data.h \
+       SMR_plot.h \
+       SMR_data_p1.h \
+       SMR_data_p2.h \
+	   	 stat.hpp \
+	     bfile.hpp \
+       SMR_data_p3.h \
+			 OPERA.h
 SRC = SMR.cpp \
-           CommFunc.cpp \
-           SMR_data.cpp \
-	   dcdflib.cpp \
-           StatFunc.cpp \
-           StrFunc.cpp	\
-           SMR_plot.cpp \
-           SMR_data_p1.cpp \
-           SMR_data_p2.cpp \
-	   stat.cpp \
-	   bfile.cpp \
-           SMR_data_p3.cpp 
+      CommFunc.cpp \
+      SMR_data.cpp \
+	    dcdflib.cpp \
+      StatFunc.cpp \
+      StrFunc.cpp	\
+      SMR_plot.cpp \
+      SMR_data_p1.cpp \
+      SMR_data_p2.cpp \
+	    stat.cpp \
+	    bfile.cpp \
+      SMR_data_p3.cpp \
+			OPERA.cpp
 OBJ = $(SRC:.cpp=.o)
 
 all : $(OUTPUT) 

